mitodl/ol-keycloak
# Implement feedback on UX

### What are the relevant tickets?
Closes #45 

### Description (What does it do?)


- Updates the theme itself to address mostly wording changes.
- Styles the profile information page.

### How can this be tested?

- Verify the wording changes are present
- Test the profile information page:
  - In the keycloak admin, create a user with blank values for first and last name, but mark them verified.
  - Attempt to login as them, you should be prompted to fill in the name fields and it should look like the screenshot.:

![Screenshot 2024-04-01 at 17-05-01 MIT Open](https://github.com/mitodl/ol-keycloak/assets/28598/8d7c6918-f288-4637-8c71-c802789ccd16)
